CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contributing to Liberty Config Language Server
```

Comment:
I don't know if this info fits what is usually in the CONTRIBUTING document. It's useful info for someone contributing, but it should be more about what the process is to [contribute](https://github.com/OpenLiberty/ci.maven/blob/main/CONTRIBUTING.md). Maybe call this document BUILDING or DEVELOPING?

Reply:
Will we need things like the Contributor License Agreement in the contribute page you linked?

Reply:
> Will we need things like the Contributor License Agreement in the contribute page you linked?

Check with YK, but I think the answer will be yes.